### ISSUE TITLE
Review concept default_variant_in_sample_schema.yaml in develop

### ISSUE THREAD

**Issue by mbaudis** [ga4gh-beacon/specification-v2-default-schemas#18]
The `default_variant_in_sample_schema` is very odd - it should *only* contain the minimal information from a single observation (i..e. from a given sample) - `referenceName`, `start`, `end`, `variantType` etc., and additionally housekeeping and linking/identifier information.

However, parameters like `PhenotypicEffect` or frequency information do not belong there - these are annotations associated with the variant prototype, not a given instance.

**Comment by jrambla** [comment]
Variant_in_sample is hosting anything that the genomics clinician consider should be highlighted, annoted, vs the "canonical"/prototype variant annotations. We have heard several times from our clinical users that they "conclude" something different of what they find in the knowledges bases and that such "discrepancies" are of highest interest for them.
Having said so, if "copy & paste" have been streched too much, we need to revisit it.

**Comment by mbaudis** [comment]
O.k. But then the naming is odd - it is a "variant report" mixing the instance of an observation with evidences & individual interpretation. This is really messy - while such a report can be generated, it then should be just an aggregator etc. There should be probably a way to have some option for annotations for a specific variant observation, but an observed variant IMO is just a bare-bones representation (position, type, sequences, some QC) of an altered allele (not even haplotype). I'm also a bit concerned that the schemas start to explode out of manageability ...

So: Keep a basic "variant_observation" or such (`allelic_variant_observation` or simply `variant`) and have a clean aggregator report, interpretation schema.

**Comment by clauw87** [comment]
I agree with mbaudis, maybe separate the variant report part (the clinician/researchers interpretation im their study) from the actual raw data of that variant instance that is intuitively belongs to the object Variant in Sample (ids, run and analysis data and that kind of things) and making the interpretation as maybe some instance of Variant Interpretation (which could include separately what it has now, interpretation/annotations by know resources and this other kind of interpretation by the study researchers..which would need different tags of the sort of 'evidence type' or 'source')

**Comment by jrambla** [comment]
IMU, what our clinician users said is that the interpretation is *just for that sample* not for every sample, therefore the "variant_in_sample" would be still valid.
I don't see that comments neither Michael or Claudia are challenging this, isn't it?
As I've said in my previous comment, if the naming or the description is misleading, we should review it.
Any other solution like moving part to the annotations could be explored, but It seems more confusing to me when "building the report", unless we duplicate the annotations in both schemas.

**Comment by mbaudis** [comment]
Well, "variant_in_sample" just sounds odd. Either it is jaust a `variant` - but this is being squatted on by `g_variant` ... as not being a specific observation but a kind of "prototypical variant" with **instances** being observed.

Just for you - I hope this doesn't get public - the whole concept of seeing a variant in a data collection as anything but **the observation in a sample using some methods** is ridiculous. Aka VCF.

The "we store primary observations of variants as calls of for this variant in a sample" concept is not tenable, scalable. There should be a strict separation between an observed genotypic morsel and its interpretation.

So a "variant" should always be an allelic (or quantitative... etc. like CNV) observation from a single experiment/sample ... And you can attach all technical, circumstantial information to it.

In contrast to this, there are the "variant_prototypes", e.g. providing information  about what knowledge has been assembled regarding "identical" variants (or, more precisely, "equivalent" variants). So ClinGen, your "line in VCF" variants etc.

The model should be to clearly match against variant instances (e.g. de-normalized VCFs w/ single sample, single allele variants) and then do secondary evidence matching. Or the other way; query prototypes & get the instances.

So, if I could, I would just make variant == variant_instance (a.k.a. "variant_in_sample" which is ... not nice). 

Doesn't change a thing (well) for querying since going by position, variantType ... etc. you match the same connected data. Just the delivery is different.

But anything else is just ... hurting. I guess with a clean model - which we don't have here yet! - life could be much easier.

**Comment by mbaudis** [comment]
Another option would be to declare the instance, observation ... anything worthwhile noting down as genomic observation to be a 

#### `genomicObservation`
OR
#### `genomicCall`

This wouldn't even make a claim about a "variant" status; just that this has been observed. And then we could leave `variant` as something with an annotation/interpretation baggage.

@ahwagner ???

**Comment by jrambla** [comment]
I don't know if I'm following you.
Let's keep the names apart for a while... IMU the concepts we are discussing are there:

1. Variant (or g_Variant) should be the prototype
2. VariantsInterpretation should be  the general annotations that could be found in the knowledge basis
3. VariantsInSample should be  the specific list of variants found in one sample (and one analysis) and the specific annotations done to it, when they depart from the canonical ones.

IMU, also this is what do you mean by:

> "The Beacon model should be to clearly match against variant instances (e.g. de-normalized VCFs w/ single sample, single allele variants) and then do secondary evidence matching. And also the other way; query prototypes & get the instances. But clearly modeled as such."

The current model tries to cover the two main use cases we are having:

1. "Clinical reports" where the person/sample/diagnosis is the core
2. "Knowledge Bases" where the accumulated knowledge is the core and they have no specific individuals or samples on it

IMU the current model is covering both and clear enough in that respect, but happy to hear and discuss any different view or question about it.

_NOTE: We are discussing many topics in the same issues and that makes it hard to follow, I suggest to open different issues for different questions (e.g. "should we show frequencies or not?"). Otherwise many questions are buried in more conceptual conversations._

**Comment by mbaudis** [comment]
> 1. Variant (or g_Variant) should be the prototype
> 2. VariantsInterpretation should be the general annotations that could be found in the knowledge basis
> 3. VariantsInSample should be the specific list of variants found in one sample (and one analysis) and the specific annotations done to it, when they depart from the canonical ones.

```
|-----------------|     |------------------|
| variantInstance |-----|                  |
|-----------------|     |     variant      |
                     |--|  * has variantId |
|-----------------|  |  |                  |
| variantInstance |---  |------------------|
|-----------------|

# Associated data (referenced or part of entity)

* technical data        * variant interpretation
  - assay, run ...        - pathogenicity
* case level data         - population frequency (if)
                          - translational effect ...

```

O.k.; that is the

> ... flipped - `variant` could be the "prototype (rsid and all), and the primary variant is then `variant_instance`. No problem with that - just be clear about it.

... from above. We just have to be clear about it - up to now, in my interpretation `g_variant` was ambiguous. 

Also: Given this, `variantInSample` expresses the right concept but sounds odd && refers to another entity in its name. Any of
* `variantInstance`
* `genomicObservation`
* `genomicCall`
... seem (to me) better suited since they just refer to the given observation itself (`variantInstance` already includes a claim but this is o.k. when speaking about ... variants).

Btw.: VRS does not use "variant" but ["variation" as the general concept](https://vrs.ga4gh.org/en/latest/terms_and_model.html#variation) which is "A state of a molecule at a Location." and does **not** imply a transition (from "normal" to "not so normal").

**Comment by mbaudis** [comment]
@jrambla O.k., actually back to start: No matter how this is named, "phenotype" etc. is part of the `individual` in whom this `variantInstance` ... was observed, and the object itself should have minimal properties:

* technical in run, analysis ...
* clinical in individual, biosample, cohort ...
* assumed phenotypic effect in the private part of a "variantAnnotation" - this could be similar to the ClinGen etc. data but with local provenance

Any variant with an _interpretation_ becomes a "prototype" which assertions related to it, based on the observed instance, clinical data and external information.

**Comment by jrambla** [comment]
We are aiming to simplify considerably in Beacon v2 Model draft 4. Please, let's continue discussions there.
